src/Illuminate/Encryption/Encrypter.php

```diff
@@ -33,16 +34,23 @@ class Encrypter implements EncrypterContract
      * Create a new encrypter instance.
      *
      * @param  string  $key
-     * @param  string|null  $cipher
+     * @param  string  $cipher
      * @return void
      */
-    public function __construct($key, $cipher = null)
+    public function __construct($key, $cipher = 'AES-128-CBC')
     {
-        $this->key = (string) $key;
+        $len = mb_strlen($key = (string) $key, '8bit');
```

Comment:
Adding it into 5.0 would just cause composer to resolve an earlier version before we added the dependency.